### PR TITLE
Beta 3

### PR DESCRIPTION
**Pull request by shornali**
please check if everything issokay and addeed

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -1009,6 +1009,379 @@ glPushMatrix();
 
 
    glPopMatrix();
```
Comment (tiham-md-inkiad) [inline main.cpp:1011]: there is some error issues please fix those first then create a pull request

**Review by tiham-md-inkiad — COMMENTED** [review]
fix error please

File: main.cpp (hunk 1 above)
Comment (shornali) [reply main.cpp:1011]: okay on it

Comment (tiham-md-inkiad) [reply main.cpp:1011]: tnx

Comment (shornali) [reply main.cpp:1011]: fixed

**Pull request merged**